back/engines/commercial/multi_tenancy/app/services/multi_tenancy/templates/serializers/custom_field.rb

```diff
@@ -16,6 +16,7 @@ class CustomField < Base
           maximum
           maximum_label_multiloc
           minimum_label_multiloc
+          select_count_enabled
```

Comment:
Is there a reason to not include maximum_select_count and minimum_select_count here?